### PR TITLE
pages.pt_BR: update outdated pages

### PR DESCRIPTION
**Pull request by sebastiaanspeck**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).

### CODE REVIEW

File: pages.pt_BR/common/cargo-rustc.md [hunk 1]
```diff
@@ -5,7 +5,7 @@
 
 - Compila o pacote ou pacotes definidos pelo arquivo `Cargo.toml` no diretório de trabalho atual:
```
Comment (vitorhcl) [inline pages.pt_BR/common/cargo-rustc.md:6]:
```suggestion
- Compila o pacote e passa opções para `rustc`:
```

File: pages.pt_BR/common/mongo.md [hunk 2]
```diff
@@ -5,15 +5,15 @@
 
 - Conecta a uma base de dados:
```
Comment (vitorhcl) [inline pages.pt_BR/common/mongo.md:6]:
```suggestion
- Conecta a uma base de dados local na porta padrão (mongodb://localhost:27017):
```

File: pages.pt_BR/common/mongo.md [hunk 3]
```diff
@@ -5,15 +5,15 @@
...
 
-`mongo {{base_de_dados}}`
+`mongo`
 
 - Conecta a uma base de dados em um host e porta específicos:
 
 `mongo --host {{host}} --port {{porta}} {{base_de_dados}}`
 
 - Conecta a uma base de dados com um usuário específico, uma senha será pedida ao usuário:
```
Comment (vitorhcl) [inline pages.pt_BR/common/mongo.md:14]:
```suggestion
- Autentica usando, na base de dados especificada, o nome de usuário especificado (uma senha será solicitada):
```

File: pages.pt_BR/common/[.md [hunk 4]
```diff
@@ -6,7 +6,7 @@
 
 - Testa se uma determinada variável é igual a uma determinada string:
```
Comment (vitorhcl) [inline pages.pt_BR/common/[.md:7]:
```suggestion
- Testa se uma determinada variável é igual a/diferente de uma determinada string:
```

**Review by vitorhcl — DISMISSED** [review]
Thanks for your contribution, I continued some of your changes. Maybe later I'll add the new examples on the files you changed.

**Comment by vitorhcl** [comment]
> @vitorhcl @isaacvicente any suggestions for the mentioned pages? Would you be so kind to review this PR as well.

I will be traveling this week, so I don't know if I can do it.

There are some removals that are, in reality, updates. I think it would be nice to split these kinds of PRs in updates and removals that needs a replacement

**Comment by vitorhcl** [comment]
> If it helps to review, I can definitely do that. Right now I have several PRs open per language. I don't know if it will do any good if I open a new PR per language to only address the removals that need replacement.

The split allows a merge of the parts I've already reviewed and modified. It's up to you, if you think that this is worth it.

**Comment by vitorhcl** [comment]
> @vitorhcl does this one look good to you?

LGTM, let's wait @isaacvicente's review.

**Review by isaacvicente — APPROVED** [review]
I can not review this as of this week so I'm gonna trust @vitorhcl 's review.

**Pull request merged**